std/experimental/allocator/building_blocks/affix_allocator.d

```diff
@@ -402,6 +402,20 @@ struct AffixAllocator(Allocator, Prefix, Suffix = void)
     });
 }
 
+// Test empty
+@system unittest
+{
+    import std.experimental.allocator.building_blocks.bitmapped_block : BitmappedBlock;
+    import std.typecons : Ternary;
+
+    auto a = AffixAllocator!(BitmappedBlock!128, ulong, ulong)
+                (BitmappedBlock!128(new ubyte[128 * 4096]));
+    assert((() pure nothrow @safe @nogc => a.empty)() == Ternary.yes);
```

Comment:
Since the unit test is explicitly marked as @system, it would probably have been better to put these in their own unit test block and mark it pure nothrow @safe @nogc.